src/features/MozAttributeComparison.ts

```diff
@@ -271,6 +353,14 @@ class MozAttributeComparison extends LitElement {
         <div class="container">
           <div class="controls-section">
             <div class="field">
+
+              <label class="label">User intent (buy strollers):</label>
```

Comment:
you can use `placeholder` to show something in the box instead of part of the label